website/content/docs/nia/license.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Consul-Terraform-Sync Enterprise License
+description: >-
+  Consul-Terraform-Sync Enterprise License
+---
+
+# Consul-Terraform-Sync Enterprise License
+<EnterpriseAlert>
+  Licenses are only required for Consul-Terraform-Sync Enterprise
+</EnterpriseAlert>
+
+Consul-Terraform-Sync Enterprise binaries require a license in order to start. The required license must be a [Consul Enterprise license](/docs/enterprise/license/overview) as there is no specific Consul-Terraform-Sync Enterprise license. As a result, Consul-Terraform-Sync Enterprise's licensing is very similar to Consul Enterprise. Consul-Terraform-Sync Enterprise is available with all its features with any valid, unexpired Consul Enterprise license regardless of its packaging/pricing model or level.
+
+To get a trial license for Consul-Terraform-Sync Enterprise, you can sign-up for the [trial license for Consul Enterprise](/docs/enterprise/license/faq#q-where-can-users-get-a-trial-license-for-consul-enterprise).
+
+## Setting the License
+
+Consul-Terraform-Sync Enterprise binary is only offered without a built-in license. Depending on how you would like to set the license, have the license value at hand or saved in a file to disk.
```

Comment:
I don't think it's necessary to mention prior ways of licensing (built in), since this will be the first and only method supported for the new enterprise project.

```suggestion
Depending on how you would like to set the license, have the license value at hand or saved in a file to disk.
```